Test for github OS changes

baltzell opened the pull request.

baltzell committed:
roll back

c-dilks commented:
is this testing the cvmfs CI failure? Ubuntu v24 is _supposed_ to be working, they test it upstream. This may be an issue in the action itself

c-dilks committed:
ci: re-trigger, this time with cache hit

the first CI run was a cache miss; let's see if things are different
with a cache hit

c-dilks commented:
Cache hit also works. So rolling back the runner's OS version fixes this, but is a kluge. Let me try flushing the CVMFS cache for Ubuntu 24; hopefully that fixes things, otherwise I'll open an issue in `github-action-cvmfs`.

c-dilks commented:
Yup... just needed a cache flush. I think we should be good now, so I'm gonna close this.

https://github.com/JeffersonLab/coatjava/actions/runs/16198428834
